Lombiq.Hosting.Azure.ApplicationInsights/Services/TelemetryFilter.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.ApplicationInsights.Extensibility;
+using System.Collections.Generic;
+
+namespace Lombiq.Hosting.Azure.ApplicationInsights.Services;
+
+public class TelemetryFilter : ITelemetryProcessor
+{
+    private readonly List<string> _errors = new()
+    {
+        "Azure.RequestFailedException: The specified container already exists.",
```

Comment:
This needs to be specific to the app's configured media or data protection container.